src/java.base/share/classes/jdk/internal/util/random/RandomSupport.java

```diff
@@ -725,11 +725,7 @@ public static float boundedNextFloat(RandomGenerator rng, float bound) {
 
     // The following decides which of two strategies initialSeed() will use.
     private static boolean secureRandomSeedRequested() {
-        @SuppressWarnings("removal")
-        String pp = java.security.AccessController.doPrivileged(
-                new sun.security.action.GetPropertyAction(
-                        "java.util.secureRandomSeed"));
-        return (pp != null && pp.equalsIgnoreCase("true"));
+        return Boolean.getBoolean("java.util.secureRandomSeed");
```

Comment:
Not sure why access to this system property required privileges.

It is documented in [SplittableRandom](https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/util/SplittableRandom.html#java.util.secureRandomSeed), so I guess it's OK to have unprivileged access.

Reply:
Previously, all properties were enabled only if allowed by the security policy. 
Any property could be denied. Though it could/would cause havoc if used indiscriminately.